### ISSUE TITLE
Output floder

### ISSUE THREAD

**Issue by kxxc** [c0decracker/video-splitter#14]
Hello, thanks for your hard working,

I'm wondering, could this py make a different video file output floder?

thanks!

**Comment by c0decracker** [comment]
Hello,

Do you mean can the output video be placed in a different folder? 
Yeah I think its a good idea. I'll add that feature in after merging the two outstanding PRs :)

**Comment by MatteoGioioso** [comment]
Hello. Has this feature been implemented? If not I could take a look and try to make a PR
Thanks

**Comment by shiro789** [comment]
hello, is this feature already added?

**Comment by c0decracker** [comment]
@shiro789 No, not yet. Happy to merge the PR if you want to try it or I'll post an update here once I get to it (eventually... :/ )

**Comment by carter300** [comment]
Push for this feature
